web/src/components/vis/index.js

```diff
@@ -0,0 +1,28 @@
+import WilcoxonPlotTile from './WilcoxonPlotTile.vue';
```

Comment:
Can you explain why this file is the `index.js` for the `vis` directory?

Reply:
cc @sgratzl

Reply:
see discussion whether this should be put in an `analysis.js` file

Reply:
sorry, I've moved that to `vis/analyses.js` now